specs/Standards/EventsFormat.md

```diff
@@ -0,0 +1,115 @@
+# Events Format ([NEP-297](https://github.com/near/NEPs/issues/297))
```

Comment:
Looks like you merged master into this branch instead of rebasing on top of that. It may confuse the readers and it gives the changes that people don't need to check

Reply:
yep, let me rework these merges thanks. Just wanted to ack that i'll rebase things to un furl this